jmsSource/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <Logger name="io.vantiq.extsrc.jmsSource" level="debug"/>
+        <Logger name="io.vantiq.extjsdk" level="debug"/>
+        <Root level="error">
+            <AppenderRef ref="Console"/>
```

Comment:
To help "some folks", would it make sense to add some appender here that writes to a file that it truncates on each start?  Then, there'd be a record.  

Maybe define it but comment it out, making it easier for anyone to use it.